clients/unshaded/src/main/java/tachyon/client/file/options/GetStatusOptions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.file.options;
+
+import tachyon.annotation.PublicApi;
+
+/**
+ * Method options for getting the status of a path
+ */
+@PublicApi
+public final class GetStatusOptions {
+  /** Whether or not to check the ufs if the path does not exist in Tachyon */
```

Comment:
If the path is in Tachyon we will just return the `URIStatus` we can get from the master. If the path doesn't exist, we can either throw an exception or attempt to fetch it from the under storage (if it is mounted) which is what this flag is for.